src/qtt/algorithms/awg_to_plunger.py

```diff
@@ -239,19 +221,19 @@ def plot_awg_to_plunger(result, fig=10):
     angle = result['angle']
 
     ds = get_dataset(result)
-    _, tr = qtt.data.dataset2image(ds)
-    xscan = tr.pixel2scan(np.array([[0], [0]]))
+    v = ds.default_parameter_array()
+    dy = v.set_arrays[0][-1]-v.set_arrays[0][0]
+    centre = [np.mean(v.set_arrays[1]), np.mean(v.set_arrays[0])]
 
-    plt.figure(fig)
-    plt.clf()
-    MatPlot(ds.default_parameter_array(), num=fig)
+    plot_dataset(ds, fig=fig)
     if angle is not None:
-        rho = -(xscan[0] * np.cos(angle) - np.sin(angle) * xscan[1])
+        rho = -(centre[0] * np.cos(angle) - np.sin(angle) * centre[1])
 
-        for offset in [-40, -20, 0, 20, 40]:
+        for offset in dy*np.linspace(-1, 1, 7):
```

Comment:
Shouldn't this change also be done in line 158 of analyse_awg_to_plunger?

Reply:
No, the image scale is different there. I did rename `offset` to `offset_in_pixels` to make this more clear